### PR TITLE
Move server to its own repository

### PR DESCRIPTION
**Pull request by mmattel**
Glory, this PR moves the server part into its own repository `docs-server`.

Docs is now the main repo including all the content sources !

Following changes have been made:

* Setup the `docs-server` repo so it contains all three server manuals from all branches 1:1
* Fix some leftover family resource identifiers in the `docs-server` repo
* Setup the `docs-server` repo in the way as all the other content sources (like desktop, ios, android ect) are
made `.drone.star`, `antora.yml`, `site.yml` ect.
* Setup the `docs` repo to handle the new content source `docs-server`
* Improve `site.yml` for better readability
* Adopt the helper scripts in the `bin/` folder
* Add a new navigation to access all the content sources directly from the main menu.
Note that accessing the latest content source version is done automatically using attributes
* Fix some leftover family resource identifiers
* Fix accessing the server stuff from releases and release-notes
* Cleanup of `.drone.strar` 

Post merging this, we can remove the 10.9 and 10.8 releases from this repo = no backport necessary.

Note that no URL's are changing 😄 

Note that this repo does not contain versioning (branches) anymore - not necessary as all the versioning is made in the component sources. When an attribute in `site.yml` regarding a `latest-xxx-version` gets changed, the link to the actual component is set automatically.

Note, this now enables us to move eg the user manual into its own repo (like the server) or integrate ocis or migrate the branding stuff into the clients.

Note that running a build is successful 😅 

**Here is an image of the new navigation**
image

@pmaier1 fyi (note that you have to use the full resource identifyer when linking stuff to server in the release notes. Just see the examples in the file and you get used to it easily. Questions welcomed)

@xoxys can you please double check here and in the `docs-server` repo the changes in `.drone.star`.

### CODE REVIEW

File: docs/antora-site-structure.md [hunk 1]
```diff
@@ -39,8 +39,9 @@ Our setup is made in a way, where each repo can build it's own content individua
 ```
 main        content source
 
-            docs-client-desktop
-docs    --> docs-client-ios-app 
+            docs-server
+docs    --> docs-client-desktop
```
Comment (phil-davis) [inline docs/antora-site-structure.md:43]:
```suggestion
docs   --> docs-server
            docs-client-desktop
```

File: modules/ROOT/pages/server_release_notes.adoc [hunk 2]
```diff
@@ -950,10 +958,10 @@ This undesired behavior is fixed with 10.2.1.
 * **Fixed issue with Sharing API and enforced public link expiration dates** +
 An issue in the Sharing API caused the ownCloud clients to prevent users from creating public links when the option "Enforce expiration date" for public links is in use. This is now solved.
 * **Fixed known issue with user avatar paths** +
-Version 10.2.0 xref:server_release_notes.adoc#known-issues[accidentally changed the location of user avatars] making them unavailable and storing uploaded avatar images in the wrong location.
+Version 10.2.0 xref:server_release_notes.adoc#known-issues-10-6[accidentally changed the location of user avatars] making them unavailable and storing uploaded avatar images in the wrong location.
```
Comment (phil-davis) [inline modules/ROOT/pages/server_release_notes.adoc:961]:
```suggestion
Version 10.2.0 xref:server_release_notes.adoc#known-issues-10-2[accidentally changed the location of user avatars] making them unavailable and storing uploaded avatar images in the wrong location.
```

File: modules/ROOT/partials/nav.adoc [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+* xref:how_to_contribute.adoc[How to Contribute]
+
+// note, atm we cant include an existing component navigation via eg
+// include::{latest-server-version}@server:ROOT:nav$partials/nav-server.adoc
+// for details about how to setup the nav link properly the see:
+// https://antora.zulipchat.com/#narrow/stream/282400-users/topic/Include.20partial.20with.20ROOT.20module.20errors 
+// BUT: the content gets presented, the links do not work - therefore not used
+// a soulution needs to be considered if the current layout should be changed
+
+.ownCloud Documentations
```
Comment (phil-davis) [inline modules/ROOT/partials/nav.adoc:15]:
```suggestion
.ownCloud Documentation
```

IMO Documentation is a group word already and covers the general concept of one and/or more documents.

**Review by phil-davis — APPROVED** [review]
Looks OK AFAICT - maybe you/we will only really know after everything is merged and published.

**Comment by xoxys** [comment]
IMO, there is a lot in the starlark file that need to be cleaned up... There are more PDF pipeline steps not commented out, the downstream trigger should not be used anymore and there is only a single `*_branch` variable required now. Only `deployment_branch` is used, the rest can be removed. I would also recommend removing unused parts instead of just commenting it to avoid cluttering. Git has a searchable history, you don't need to keep a backup on your own.

**Comment by mmattel** [comment]
> IMO, there is a lot in the starlark file that need to be cleaned up

@xoxys can you help? My knowledge about this file is very limited and I do not want to break things...

**Comment by xoxys** [comment]
See my commit. Removed everything that is not used anymore.

File: antora.yml [hunk 4]
```diff
@@ -1,16 +1,9 @@
-name: server
-title: ownCloud Server
+name: docs
+title: ownCloud Docs
```
Comment (EParzefall) [inline antora.yml:2]:
```suggestion
title: ownCloud Manuals
```

**Pull request merged**